incense/experiment.py

```diff
@@ -143,7 +143,10 @@ def __getattr__(self, item):
     def from_run_dir(cls, run_dir: Path):
         id_ = int(run_dir.name)
         config = _load_json_from_path(run_dir / "config.json")
-        info = _load_json_from_path(run_dir / "info.json")
+        if "info.json" in [f.name for f in run_dir.iterdir()]:
```

Comment:
I think we could simply check if the path [exists](https://docs.python.org/3/library/pathlib.html#pathlib.Path.exists) and only read it then instead of iterating over the directory.

Reply:
Good point